ecaudit/src/test/java/com/ericsson/bss/cassandra/ecaudit/logger/TestSlf4jAuditLogger.java

```diff
@@ -54,7 +54,7 @@
 @RunWith(MockitoJUnitRunner.StrictStubs.class)
 public class TestSlf4jAuditLogger
 {
-    private static final String DEFAULT_LOG_FORMAT = "client:'${CLIENT_IP}'|user:'${USER}'{?|batchId:'${BATCH_ID}'?}|status:'${STATUS}'|operation:'${OPERATION}'";
+    private static final String DEFAULT_LOG_FORMAT = "client:'${CLIENT_IP}'{?|user:'${USER}'?}{?|batchId:'${BATCH_ID}'?}|status:'${STATUS}'|operation:'${OPERATION}'";
```

Comment:
is the user optional?

Reply:
Leftover from bad interactive rebase, fixing.